src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,82 @@
         </div>
     {% endif %}
 
+    <h2>Security Firewall</h2>
+
+    {% if collector.firewall|default([]) is not empty %}
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name }}</span>
+                <span class="label">Name</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.security ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Security</span>
```

Comment:
Might be a bit too long but what about:
 - Authentication required
 - Security enabled
 - Requires authentication